### DISCUSSION TITLE
workflow 8-2-3 issue's

### DISCUSSION THREAD

**Discussion by ari2asem** [R3S3t9999/DoVi_Scripts#36] [General]
![image](https://github.com/user-attachments/assets/e2b1170b-e80a-447f-861c-3d83344e7124)
i get issue's with workflow 8-2-3.

i got python 3.12 and any other tools installed as instructed.

when using workflow 3-1, i dont get errors about temp.folder. and also tried v3 of the script. with still same error

another question: how long does it take for madvrmeasurement when working with 3-1 workflow? i am working with oppenheimer movie (hdr10) to convert to dovi-version.

**Comment by R3S3t9999** [comment]
> i get issue's with workflow 8-2-3.

Well according to your picture, you didn't drag and drop a file. 
I use 8-2-3 every day so there is no issue that I'm aware of.

> another question: how long does it take for madvrmeasurement when working with 3-1 workflow? i am working with oppenheimer movie (hdr10) to convert to dovi-version.

it takes about 5-10minutes on my rtx3070 with lavfilter D3D11 (native). The slow performance issue with 3-1 usually happens with dolby cm_analyze.exe and I have no control on it. You will get much better speed if you do it [manually in Resolve.](https://www.youtube.com/watch?v=FVSh3oGqfXY)

**Comment by ari2asem** [comment]
well, first things first.

the error in the screenshot was caused because the movie file was on different disk. i was running the dovi-script from disk Y, movie file was on disk F.
after transfering the whole folder of the script (including TOOLS),  i could start running the script.

well, i took 4 days running the workflow 3-1 (making DV fmovie from HDR10 movie). my system is AMD 3400g. i know that my system is slow for this kinid of job.

but what i saw during 3-1 workflow, is the weird behaviour of madvr hdr measurement.  i don't if it is weird.
but hdr measurement needs user (that would me) interaction to function properly. 

in the taskbar i need to click on madvr icon, then click START and PAUSE, then again START to get proces (hdr measurement) starting in your script.
is this normal behaviour of madvr hdr measurement?

2nd question....i ran 3-1 workflow. now i have DV P8.1 movie. i thougth i will get DV P7 FEL movie.
is the output of 3-1 always P8.1?
if yes, maybe it is better to see that somewhere in your script (like as warning/announcement ).

but after all.....thanks for your all efforts , doing great job for DV-fans  :-)

**Comment by R3S3t9999** [comment]
> the error in the screenshot was caused because the movie file was on different disk. i was running the dovi-script from disk Y, movie file was on disk F.

That is not possible sorry. Your picture clearly shows no input and I do encoding from different drives every day.

>is this normal behaviour of madvr hdr measurement?

No, not normal and it never happened to me or any other user so I don't know what's causing this on your end...

> 2nd question....i ran 3-1 workflow. now i have DV P8.1 movie. i thougth i will get DV P7 FEL movie.
is the output of 3-1 always P8.1?
if yes, maybe it is better to see that somewhere in your script (like as warning/announcement ).

No warning is needed.  Profile 7 is a dual-layer format from bluray disc which cannot be created without the original Master.
Profile 8 is 1:1 the same quality as P7 MEL anyway  and since we don't have the 12-bit master to create FEL P7, there is no point in generating Profile7 MEL... Profile 8 also has better support on devices.
